### PR TITLE
Fixed netcore tests

### PR DESCRIPTION
**Pull request by ncave**
The Regex not being replaced is still an issue, so those tests are disabled for now.

### CODE REVIEW

File: src/tests/fableconfig.json [hunk 1]
```diff
@@ -10,9 +10,8 @@
     "plugins": ["../plugins/nunit/Fable.Plugins.NUnit.dll"],
     "targets": {
         "netcore": {
-            "projFile": "project.fsx",
+            "projFile": ["Main/project.fsx"],
             "ecma": "es2015",
-            "module": "commonjs",
```
Comment (alfonsogarciacaro) [inline src/tests/fableconfig.json]: Why is this deleted? The default module system now is `es2015`. It won't be possible to run the tests on Mocha if `commonjs` is not used.

File: src/fable/Fable.Client.Node/js/fable.js [hunk 2]
```diff
@@ -386,8 +386,9 @@ function readFableConfigOptions(opts) {
         var projDir = fs && fs.statSync(fullProjFile).isDirectory()
                         ? fullProjFile
                         : path.dirname(fullProjFile);
-        cfgFile = fableLib.pathJoin(projDir, constants.FABLE_CONFIG_FILE);
-
+        if (fs && !fs.existsSync(cfgFile)) {
+            cfgFile = fableLib.pathJoin(projDir, constants.FABLE_CONFIG_FILE);
+        }
```
Comment (alfonsogarciacaro) [inline src/fable/Fable.Client.Node/js/fable.js]: Not sure this is a good idea. For example in [fable-powerpack](https://github.com/fable-compiler/fable-powerpack) there's a fableconfig file in the root dir and in the tests dir. If I call (from root) `fable tests` I expect the fableconfig file in `tests` directory to be read, not the one in the current (root) dir. That will fail with this change, can you please revert it?

**Comment by alfonsogarciacaro** [comment]
Why is Regex not being replaced? Those tests worked before, right?

**Comment by ncave** [comment]
@alfonsogarciacaro Ok updated with your suggestions.
Regex replacement works in master, but not 0.7.0, that's why it's disabled for now so
`build FableCompilerNetcore` can pass.

**Comment by alfonsogarciacaro** [comment]
Thanks! Strange, I think I haven't touched the Regex tests during 0.7.0, I'll need to have a look.

**Comment by ncave** [comment]
@alfonsogarciacaro I don't think it's the tests themselves, perhaps it has something to do with references, not sure, anyway, if you uncomment the tests in project.fsx you'll see the issue.
